v2/go.mod

```diff
@@ -2,4 +2,10 @@ module github.com/googleapis/gax-go/v2
 
 go 1.11
 
-require google.golang.org/grpc v1.19.0
+require (
+	github.com/google/go-cmp v0.5.6
+	google.golang.org/genproto v0.0.0-20210715145939-324b959e9c22
+	google.golang.org/grpc v1.39.0
+	google.golang.org/protobuf v1.27.1
```

Comment:
@codyoss are we OK with these updates? Just want to make sure since this is a core module.

Reply:
Just left one small comment, but I am good with the rest of the changes